### PR TITLE
[luci] Check more validation for BCQ information

### PR DESCRIPTION
**Pull request by llFreetimell**
Parent Issue : #4337
Full Draft : #4338

This commit will add more validation for BCQ information

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

File: compiler/luci/pass/src/FuseBCQPass.cpp [hunk 1]
```diff
@@ -267,13 +267,124 @@ template <> class BCQFuser<1>
...
+      {
+        LOGGER(l);
+        WARN(l) << "FuseBCQPass : alpha has wrong type" << std::endl;
+        return false;
+      }
+
+      if (ref_prefix_set.find(n.first) == ref_prefix_set.end())
+      {
+        LOGGER(l);
+        WARN(l) << "FuseBCQPass : Invalid prefix is found" << std::endl;
```
Comment (seanshpark) [inline compiler/luci/pass/src/FuseBCQPass.cpp:299]: Q) from the condition, is the meaning like `Required prefix not found. Invalid file` ?

Comment (llFreetimell) [reply compiler/luci/pass/src/FuseBCQPass.cpp:299]: At first, I wanted to check that `prefix` is valid when we access to `map`.
However, it looks weird to add the assertion in this function.
I will remove this check in this PR and create another PR for it :)

**Review by seanshpark — APPROVED** [review]
LGTM thank you

**Comment by mhs4670go** [comment]
@llFreetimell Just curiosity, I think the pass spits out WARNING instead of error because it shouldn't be stopped even if there are some mismatch with what it should be. But if there are some wrong condition, the converted model will be wrong model. Is it worth to generate it instead of throwing an error?

**Comment by llFreetimell** [comment]
@mhs4670go Answered at offline :)

**Pull request merged**